tycho-its/src/test/java/org/eclipse/tycho/test/baseline/BaselineMojoTest.java

```diff
@@ -55,17 +91,39 @@ public void testUnchangedApi() throws Exception {
 	 * @throws Exception
 	 */
 	@Test
-	public void testAddMethod() throws Exception {
+	public void testAddMethod() throws Throwable {
+
 		// test adding a default method to the "public" interface
-		Verifier verifier;
-		verifier = buildBaselineProject("add-method", true);
-		verifyBaselineProblem(verifier, "ADDED", "METHOD", "concat(java.lang.String,java.lang.String)", "1.0.0",
+		Verifier verifier1 = buildBaselineProject(true, r -> {
+			Path aClass = r.resolve(Path.of("src", "my", "api", "bundle", "MyApiInterface.java"));
+			modifyJavaSourceFile(aClass, cu -> {
+				TypeDeclaration apiInterface = (TypeDeclaration) cu.types().get(0);
+				addMethod(apiInterface, "concat", m -> {
+					setSimpleReturnType(m, "String");
+					addSimpleTypeParameter(m, "String", "a");
+					addSimpleTypeParameter(m, "String", "b");
+				});
+			});
+		});
+		verifyBaselineProblem(verifier1, "ADDED", "METHOD", "concat(java.lang.String,java.lang.String)", "1.0.0",
```

Comment:
I'm unsure why in the meantime (in a previous state of this PR) the baseline plugin suggested 2.0.0 instead of 1.1.0. 
But since this used to be 1.1.0 I think it is fine to leave that unchanged.